### PR TITLE
Remove on screen effect

### PR DESCRIPTION
**Pull request by slavanga**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ general bugfixes
 ## v2.5.0
 ###### DATA MIGRATIONS
 - Image icon_enabled is now False by default [#55](https://github.com/allink/allink-core/pull/55)
+- Removed onscreen effect in the allink_content plugin [#57](https://github.com/allink/allink-core/pull/57)
```
Comment (tuerlefl) [inline CHANGELOG.md]: there is no need to list this here. (also "Image icon_enabled is now False.." sorry, did't see it the last time.) we only list **data** Migrations here, which have to be done on project level manually. You can move them to NEW

Comment (slavanga) [reply CHANGELOG.md]: alright 👍

File: allink_core/core_apps/allink_content/models.py [hunk 2]
```diff
@@ -143,12 +143,6 @@ class AllinkContentPlugin(CMSPlugin):
         blank=True,
         null=True
     )
-    project_on_screen_effect = models.CharField(
```
Comment (tuerlefl) [inline allink_core/core_apps/allink_content/models.py:146]: when we remove this, we won't be able to add a "project_on_screen_effect" on project basis. Is this something we never want to do again? I would suggest that we keep the field on the content Plugin or the moment.

Comment (slavanga) [reply allink_core/core_apps/allink_content/models.py:146]: The effect could still be added on project basis using the `CONTENT_CSS_CLASSES` in `settings.py`. If we only remove the frontend part then there is an option in the CMS that does nothing.

Comment (tuerlefl) [reply allink_core/core_apps/allink_content/models.py:146]: yes true. but CONTENT_CSS_CLASSES is in my opinion less intuitiv for the CMS user. But ok, let's remove it. Less stuff to maintain, I like it :)

**Review by tuerlefl — APPROVED** [review]

**Pull request merged**